system/Email/Email.php

```diff
@@ -1317,6 +1317,8 @@ protected function buildMessage()
     }
 
     /**
+     * @param string $type
```

Comment:
Why is thos typehinted? The argument does not seem to be.

Reply:
No, I think it is correct. I was just wondering how the fixer determined it. Turns out it wasn't the fixer 😆